mesonbuild/interpreter/interpreter.py

```diff
@@ -1297,15 +1298,18 @@ def func_project(self, node: mparser.FunctionNode, args: T.Tuple[str, T.List[str
 
         self.build.projects[self.subproject] = proj_name
         mlog.log('Project name:', mlog.bold(proj_name))
-        mlog.log('Project version:', mlog.bold(self.project_version))
+        mlog.log('Project version:', mlog.bold(self.project_version or "unset"))
 
         if not self.is_subproject():
             # We have to activate VS before adding languages and before calling
             # self.set_backend() otherwise it wouldn't be able to detect which
             # vs backend version we need. But after setting default_options in case
             # the project sets vs backend by default.
             backend = self.coredata.get_option(OptionKey('backend'))
+            print(backend)
+            assert backend is None or isinstance(backend, str)
```

Comment:
With these typing asserts we usually add a message like `'for mypy'`, which is self documenting.

Reply:
Fixed, thanks! Also removed that stray print.